entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+testConnection="/opt/mssql-tools/bin/sqlcmd -S sql -U sa -P Password123"
+
+until $testConnection; do
+>&2 echo "SQL Server is starting up.."
+sleep 1
+done
```

Comment:
this seems to be a very tight loop and as David suggested may result in an infinite loop if the server startup failed for whatever reason. I would set a limit and increase the wait.
Actually there's a travis_retry can we make use of that?